src/components/@shared/FormFields/FilesInput/index.tsx

```diff
@@ -23,7 +23,12 @@ export default function FilesInput(props: InputProps): ReactElement {
         ? values?.services[0].providerUrl.url
         : asset.services[0].serviceEndpoint
       setIsLoading(true)
-      const checkedFile = await getFileUrlInfo(url, providerUrl)
+      const urlFile: UrlFile = {
+        type: 'url',
+        url: url,
```

Comment:
shorthand notation please, which then most likely allows us to have this short line in one line, like:

```js
const urlFile: UrlFile = { type: 'url', url, method: 'GET' }
```